pytg/sender.py

```diff
@@ -134,6 +134,7 @@
 # functions["channel_set_admin"]		= ("channel_set_admin", <channel> <admin> <type>  Sets channel admin. 0 - not admin, 1 - moderator, 2 - editor
```

Comment:
Maybe you can have a look at this old line here, while your on it.
I wonder why it is not implemented.